bankid/syncclient.py

```diff
@@ -150,17 +144,17 @@ def phone_authenticate(
         response = self.client.post(self._phone_auth_endpoint, json=data)
 
         if response.status_code == 200:
-            return response.json()
+            return response.json()  # type: ignore[no-any-return]
```

Comment:
Are `# type: ignore[no-any-return]` really necessary everywhere? Can the `json` method return anything else that a dict?

Reply:
It's necessary since the json method are explicitly typed as returning `Any`. `warn_return_any = true` in mypy.ini yields the  type error that I added ignores for here. We could explicitly cast it to dict instead but we might as well leave it like this until we add type annotations for responses as I mentioned.